hangman/hangman.py

```diff
@@ -38,7 +38,7 @@ def hangman():
             for val in dic_list:
                 print(val + ": {} ".format(dic_list[val]))
             break
-        elif count == times:
+        elif count == attempt:
```

Comment:
I specifically said to track FAILED ATTEMPTS.

Reply:
I don't understand
Is failed as in the wrong input?

Reply:
When they make a wrong guess, it's a failed attempt.